### ISSUE TITLE
Xcode 8 issue - JS ERROR TypeError: NSNotificationCenter.defaultCenter is not a function.

### ISSUE THREAD

**Issue by raefa** [EddyVerbruggen/nativescript-local-notifications#16]
Thanks for the very handy module!

I upgraded to Xcode 8 and Nativescript 2.3.0, and I'm using 1.1.2 of this module. 

My app is crashing as soon as the module is referenced. I tried installing your demo app with the following in the `package.jspn` file:
```
{
    "nativescript": {
        "id": "org.nativescript.localnotifications",
        "tns-android": {
            "version": "2.3.0"
        },
        "tns-ios": {
            "version": "2.3.0"
        }
    },
    "dependencies": {
        "nativescript-local-notifications": "1.1.2",
        "tns-core-modules": "2.3.0"
    },
    "devDependencies": {
        "babel-traverse": "6.8.0",
        "babel-types": "6.8.1",
        "babylon": "6.8.0",
        "filewalker": "0.1.2",
        "lazy": "1.0.11",
        "nativescript-dev-typescript": "^0.3.1",
        "typescript": "^1.8.2"
    }
}
```

The app builds and then crashes with the following error:
```
Sep 17 08:30:10 raef LocalNotifications[28139]: file:///app/tns_modules/nativescript-local-notifications/local-notifications.js:4:44: JS ERROR TypeError: NSNotificationCenter.defaultCenter is not a function. (In 'NSNotificationCenter.defaultCenter()', 'NSNotificationCenter.defaultCenter' is an instance of NSNotificationCenter)
Sep 17 08:30:10 raef com.apple.CoreSimulator.SimDevice.A02F4667-157D-4C2F-8665-8DBA15F8995C.launchd_sim[19818] (UIKitApplication:org.nativescript.localnotifications[0x9b6a][28139]): Service exited due to Segmentation fault: 11
```

This is the same error I am getting in my project. 

Any ideas?

**Comment by raefa** [comment]
If instances of `NSNotificationCenter.defaultCenter().` are changed to `NSNotificationCenter.defaultCenter.` in the localnotifications.ios.js file (the whole methods/functions thing that has changed in iOS10) then the app builds but we see `Failed to get InstanceID...` and `found no keychain client entitlements` errors in logs ... and push notifications don't work. Not sure if that help!

**Comment by alg** [comment]
There's another bit the requires a change -- https://github.com/EddyVerbruggen/nativescript-local-notifications/blob/master/local-notifications.ios.js#L4

`NSOperationQueue.mainQueue()` becomes `NSOperationQueue.mainQueue` for the same reason @raefa stated above.

**Comment by EddyVerbruggen** [comment]
Correct, and actually [this is the backward compatible way](https://github.com/EddyVerbruggen/nativescript-appversion/commit/aa6ad999e798ba5f503dbf48c42550882de32c29) to fix it. Will do shortly!

**Comment by EddyVerbruggen** [comment]
Can you guys please give the `master` branch a spin? (not on npm yet)

**Comment by alg** [comment]
@EddyVerbruggen worked like charm. Thanks, mate.

**Comment by EddyVerbruggen** [comment]
Thanks for the quick test, published 1.1.3.
